### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by deeprun** [xunhuang1995/AdaIN-style#9]
Can you share the training code?

**Comment by diggerdu** [comment]
I am reproducing this work in Tensorflow

**Comment by gsssrao** [comment]
#5

**Comment by ArturoDeza** [comment]
Has anyone succeeded on re-implementing the training procedure? I am going to give it a shot over the weekend to better understand the algorithm. Also, hopefully I understand this correctly but the only learnable network in the entire pipeline (Figure 2) is the VGG Decoder? The AdaIN is a deterministic function with no learnable parameters as stated in section 5 of the ArXiv paper? Is this correct @xunhuang1995 ? Hmm if this is the case, maybe I can start playing around with loading the DeCoder (as pre-trained network), and continue doing iterations over new style images.

**Comment by diggerdu** [comment]
@ArturoDeza 
decoder is trainable
this is my implementation
https://github.com/diggerdu/Vincent/blob/master/normalise/vincent.py

**Comment by xunhuang1995** [comment]
The training code has been updated.
